packages/plugin/src/Fields/Implementations/Pro/CalculationField.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Solspace\Freeform\Fields\Implementations\Pro;
+
+use Solspace\Freeform\Attributes\Field\Type;
+use Solspace\Freeform\Attributes\Property\Input;
+use Solspace\Freeform\Fields\Implementations\TextField;
+
+#[Type(
+    name: 'Calculation',
+    typeShorthand: 'calculation',
+    iconPath: __DIR__.'/../Icons/calculator.svg',
+    previewTemplatePath: __DIR__.'/../PreviewTemplates/calculation.ejs',
+)]
+class CalculationField extends TextField
+{
+    public const DEFAULT_CALCULATIONS = '';
+    protected string $instructions = '';
+    protected string $placeholder = '';
```

Comment:
Lets have a space between constants and variable definitions.
Also there's no need to make it public, since nothing else uses it.

```suggestion
    private const DEFAULT_CALCULATIONS = '';

    protected string $instructions = '';
    protected string $placeholder = '';
```